.github/workflows/milestone-worker.yml

```diff
@@ -69,10 +69,10 @@ jobs:
           -DgenerateBackupPoms=false \
           -Dartifactory.publish.artifacts=false \
           -DnewVersion='${parsedVersion.majorVersion}.${parsedVersion.minorVersion}.${parsedVersion.incrementalVersion}-'${BUILD_ZOO_HANDLER_milestone_version} \
-          -B
-        BUILD_ZOO_HANDLER_spring_cloud_dataflow_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout) >> $GITHUB_ENV
-        echo BUILD_ZOO_HANDLER_spring_cloud_dataflow_version=$BUILD_ZOO_HANDLER_spring_cloud_dataflow_version >> $GITHUB_ENV
-        echo "::notice ::set dataflow.version=${BUILD_ZOO_HANDLER_spring_cloud_dataflow_version}"        
+          -B        
+        BUILD_ZOO_HANDLER_spring_cloud_dataflow_version=$PROJECT_VERSION
```

Comment:
I don't think `PROJECT_VERSION` is defined until below. I could be missing something though.